tests/Routes/User/RegisterTest.php

```diff
@@ -19,20 +20,44 @@ class RegisterTest extends TestCase
 
     // TODO test password length when not deving
 
+    public function mockReCaptchaValidation($fakeResponse=[])
```

Comment:
I know you didn't like having to copypasta this from the PR description. I wondered if an alternative that would be a little smaller would be to have a mock rule that only had the `passes` method mocked rather than the extra complexity of mocking the `verify` method?

I think that would make sense since each of these tests really doesn't want to know about the internal of the Rule (incidentally making the `verifiy` method private would prevent this systematically)

In this case you could perhaps almost inline it in any given test with *something* like
```
$mockValidation = $this->createMock(ReCaptchaValidation::class);
$mockValidation->method('passes')->willReturn(true) // or false
$this->app->instance(ReCaptchaValidation::class, $mockValidation);
```

(above was slapped together sans IDE so might need a little tweaking)

Reply:
ooh that's nice!